containers/profile/statistics.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <section class="statistics padded">
+    <header>
+      <h2 class="btn-align">Statistics</h2>
+      <Dropdown class="btn-shift">
```

Comment:
Will need to be changed when #30 is merged

Reply:
Done